eng/versioning/version_client.txt

```diff
@@ -53,7 +53,7 @@ com.azure:azure-communication-common;1.0.0;1.0.1
 com.azure:azure-communication-sms;1.0.0;1.0.1
 com.azure:azure-communication-identity;1.0.0;1.0.1
 com.azure:azure-communication-phonenumbers;1.0.0-beta.7;1.0.0-beta.8
-com.azure:azure-containers-containerregistry;1.0.0.beta.1;1.0.0-beta.1
+com.azure:azure-containers-containerregistry;1.0.0-beta.1;1.0.0-beta.2
```

Comment:
Should there be a guard against this getting committed in the first place?

Reply:
It probably makes sense to have a check for this.